nslsii/utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import re
+import subprocess
+
+
+def run(cmd, path="", ignoreErrors=True, returnError=False, debug=False):
+    """cmd should be a list, e.g. ["ls", "-lh"]
```

Comment:
The docstring should have a break between the first line and the following lines. Maybe an introductory sentence about this function can be useful.

Also, `run` seems to be too vague. Maybe something more specific such as `execute_command` can be more appropriate.